app/history.py

```diff
@@ -19,44 +19,65 @@
 import app.log
 import cPickle as pickle
 import os
+import hashlib
 
-data = {}
-path = None
+fileHistory = {}
+userHistory = {}
+pathToHistory = None
```

Comment:
How about if the code that uses |pathToHistory| use app.prefs.prefs['userData']['historyPath'] instead?